Makefile

```diff
@@ -1,64 +1,56 @@
-SRCS := $(shell find . -name '*.go')
-PKGS := $(shell go list ./...)
```

Comment:
These seemed a little odd since none of the tools actually needed them. This came from a contribution/pull request. Maybe their intent was to speed things up, to only scan the file system once ? Seems totally unnecessary, and removing bash loop from the calls below makes me like the lay of the land a little more.